### ISSUE TITLE
Matreshka.js is now Seemple.js - what is going on?

### ISSUE THREAD

**Issue by finom** [finom/seemple#52]
tl;dr to upgrade
1. Use global search/replace for "matreshka" keywords: `Matreshka` -> `Seemple`, `MK` -> `Seemple` (there is no shortcut anymore), `matreshka` -> `seemple` (for imports).
2. If you use a modular environment, update your package.json to make it use latest version of `seemple` package and run `npm install`

There was plenty of issues with Matreshka.js project so far:
1. Lack of users (I can judge it by none of Google alerts for years).
2. The project didn't brought any donations but it's for years it ate money for the .io domain and for DigitalOcean server.
3. Hard to maintain relative projects such as router and parse-form function because they were placed at different repositories. Any updates was painful, even just dependency updates.
4. The name. I would ask apologies for bringing politics to what isn't supposed to be political but the name is supposed to be fun 7 years ago but now it's associated with one aggressive country ("matreshka" means "nested doll"). An author is also not from that country and never been there. I'm really sorry for saying that. I had discussions with people from USA and Canada and they just wouldn't like to use something called "matreshka" in their projects because they don't want to scare their clients. Also everybody is confused how to pronounce "matreshka" name.

There is what is going to be happened. Work in progress but I think I can finish everything in a few days.
1.  The project is renamed.
2. The organisation on Github is going to be archived.
3. The framework and relative libraries are merged into a single repository powered by Lerna. Updates  can be brought much easier after that .
4. No paid domain or server anymore. Docs are going to be hosted at Github pages and available at https://seemple.js.org/ (all languages) forever!
5. The blog in Russian on Habr isn't going to be updated any more but the old articles still going to make sense, just do the replacements explained in the beginning of this text.
6. All links to social stuff such as Twitter or Gitter are going to be removed from everywhere.

Would these updates bring any new users or would they turn away existing ones? I don't know. A new name doesn't make the project better. But at least we're going to get a maintainable framework which doesn't look like a thing made for locals by locals.
